goon/browserassets/js/browserOutput.js

```diff
@@ -171,25 +171,67 @@ function setHighlightColor(match) {
 	match.style.background = opts.highlightColor
 }
 
+function escapeRegex(input){ // put this in a function incase it ever needs to be used elsewhere, makes code cleaner
+	return input.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+}
+
 //Highlights words based on user settings
 function highlightTerms(el) {
 	if(regexHasError) return; //just stop right there ig the regex is gonna except
+	function highlightRecursor(element,term){ //recursor function to do the highlighting proper
```

Comment:
Style: rest of the file has a space after coma in argument list.

Reply:
reeeeeeeee

Reply:
don't ignore style advice and mark it as "resolved"